pkgs/development/python-modules/online-judge-api-client/default.nix

```diff
@@ -0,0 +1,35 @@
+{ buildPythonPackage, fetchPypi, git, lib, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "online-judge-api-client";
+  version = "10.10.0";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit version format;
+    pname = "online_judge_api_client";
+    dist = "py3";
+    python = "py3";
+    sha256 = "932dafa971d0cd3ebdb7444e049dc0719f87b93123af020e4d3d1bbe499e9142";
+  };
```

Comment:
I would really prefer to use the source over the wheel: https://github.com/online-judge-tools/oj